lib/glueby/contract/token.rb

```diff
@@ -73,7 +73,7 @@ def issue!(issuer:, token_type: Tapyrus::Color::TokenTypes::REISSUABLE, amount:
                            raise Glueby::Contract::Errors::UnsupportedTokenType
                          end
           txs.each { |tx| issuer.internal_wallet.broadcast(tx) }
-          new(color_id: color_id, script_pubkey: script_pubkey)
+          [new(color_id: color_id, script_pubkey: script_pubkey), txs[1]]
```

Comment:
`txs` has two transactions in the case of `issue_reissuable_token`, and one transaction in the other cases. 
So in other case, the 2nd result value will always return `nil`.